source/server-discovery-and-monitoring/tests/sharded/discover_single_mongos.json

```diff
@@ -0,0 +1,28 @@
+{
+  "description": "Discover single mongos",
+  "uri": "mongodb://a/?directConnection=false",
+  "phases": [
+    {
+      "responses": [
+        [
+          "a:27017",
+          {
```

Comment:
The driver fails this test because there's no min/max wire version in the response so it thinks the max wire version is 0, which is lower than the driver's lowest accepted (2). Looks like other tests you wrote specify 6. Can this be changed to match?